### PR TITLE
Changes to handle file/image download issues

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: Source/core/Parser.h [hunk 1]
```diff
@@ -80,7 +80,7 @@ namespace Core {
         {
             _state |= FLUSH_LINE;
         }
-        inline void PassThrough(const uint16_t passThroughBytes)
+        inline void PassThrough(const uint64_t passThroughBytes)
```
Comment (pwielders) [inline Source/core/Parser.h]: This on an embedded system will not fly. There is no way we can parse files that are > 4G (which is a uint32_t), so uint64_t is definitielt too big.. Is there a situation where you need to exceed 64K (probably for the image file?) Would uint32_t be sufficeient than ?

Comment (HaseenaSainul) [reply Source/core/Parser.h]: Thanks Pierre pointing this. I just increased to max value, I will check with uint32_t

Comment (pwielders) [reply Source/core/Parser.h]: :+1:

Comment (HaseenaSainul) [reply Source/core/Parser.h]: Changed data type to uint32_t, since its fine :)

**Pull request merged**